lib/logitech_receiver/settings_templates.py

```diff
@@ -1646,15 +1646,15 @@ class PerKeyLighting(_Settings):
     description = _("Control per-key lighting.")
     feature = _F.PER_KEY_LIGHTING_V2
     keys_universe = _special_keys.KEYCODES
-    choices_universe = _special_keys.COLORS
+    choices_universe = _special_keys.COLORSPLUS
 
     def read(self, cached=True):
         self._pre_read(cached)
         if cached and self._value is not None:
             return self._value
         reply_map = {}
         for key in self._validator.choices:
-            reply_map[int(key)] = 0xFFFFFF  # can't read so fake a value of white
+            reply_map[int(key)] = -1  # this signals no change
```

Comment:
Please use at least a const with a name. This solves many issues at once.

`NO_KEYLIGHTING = -1`